packages/docs/docs/dev/connectors.mdx

```diff
@@ -3,78 +3,75 @@ title: Wallet Connectors
+waht connectors are installed.
 
-To select a different Wallet Connector use the `selectConnector(connectorName: string)` method.
+We also recommend to use the `connectors` listener on places that will use the `connectors` method as the availability can change.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="selectConnector"
+  file="../../examples/connectors/ListConnectors.tsx"
+  commentBlock="listConnectors"
 />
+<Examples.ListConnectors />
 
-## Events
+## Selecting Connector
 
-### CurrentConnector
-
-This event is triggered when the Current Connector is updated. The callback function receives the `FuelWalletConnector` object.
+Once you have a list of connectors, you can allow the user to select the connector he want to use. By using the methoid `selectConnect`.
+If the connector is not installed, the SDK will return false.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventCurrentConnector"
+  file="../../examples/connectors/SelectConnector.tsx"
+  commentBlock="selectConnector"
 />
 
-### Connectors
+### Interacting with the selected connector
 
-This event is triggered when the Connectors list is updated. The callback function receives the list of connectors, `Array<FuelWalletConnector>`.
+Once you have selected a connector, you can interact with it using all the available methods.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventConnectors"
+  file="../../examples/connectors/SelectConnector.tsx"
+  commentBlock="connect"
 />
 
-## For Wallet Developers
-
-Third-party wallets developed by the community can use the Wallet Connector to integrate with the current `fuel` instance on the window, enabling a better experience for
-developers and users.
+<Examples.SelectConnector />
 
-### Creating a Wallet Connector
+## With React
 
-Inside `pageScript.ts` you can call the method `createConnector` instead of creating a new object on the window. This method will add your Connector to the current,
-`window.fuel` element.
+### Connectors UI
 
-```ts
-import { createConnector } from '@fuel-wallet/sdk';
+When using the react application you can use the Connectors UI provided by react package.
 
-createConnector({ name: 'Wallet Name' });
-```
+<Examples.ConnectorUIHook />
 
-If you are using our Communication SDK [Content Page Script](https://developer.chrome.com/docs/extensions/mv3/content_scripts/) import `ContentProxyConnection` and pass the name of Wallet Extension.
+You can see the full a DApp example on the [examples](https://github.com/FuelLabs/fuels-wallet/tree/master/examples) folder.
 
-> Otherwise, remember to filter events by the ones that have your Wallet Connector Name as the target.
+### Only hooks
 
-```ts
-import { ContentProxyConnection } from '@fuel-wallet/sdk';
+If you prefer to build your own UI for the connectors you can achive the experience using
```

Comment:
<img width="1512" alt="Screenshot 2023-11-23 at 10 47 07 AM" src="https://github.com/FuelLabs/fuels-wallet/assets/106365423/d17cf85e-c707-4179-a616-a713ead7ce90">
Fuelet connection opens up the Fuel Wallet instead